packages/components/src/styles/display/display.stories.ts

```diff
@@ -43,21 +35,28 @@ export const Default = {
 };
 
 /**
- * Use the `inverted` class to make a display with inverted colors.
+ * Use the `sd-display--size-` class to set the display’s size:
```

Comment:
Please align with Design.
Text here is: Use “sd-display” modifiers for alternative appearances.